src/MEDS_DEV/models/cehrbert/pretrain_cehrbert.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import subprocess
+from pathlib import Path
+
+import hydra
+from omegaconf import DictConfig, OmegaConf
+
+logger = logging.getLogger(__name__)
+
+CONFIG = Path(__file__).parent / "_config.yaml"
+pretraining_yaml_template = Path(__file__).parent / "cehrbert_pretrain_template.yaml"
+
+
+def run_subprocess(cmd: str, temp_work_dir: str, out_dir: Path) -> None:
```

Comment:
Would it be beneficial were I to abstract this somewhere? Or should we see if there is a better package for this or something? I feel like there must be... (no change needed, just flagging the repeated code).

Reply:
@mmcdermott Totally agreed, I was thinking this would be a utility function that I could call from MEDS-DEV, and I was gonna change the models.__main__ function, but I realized this would be another chicken and egg problem. I think I would have to create a separate PR to add this common utility function, I am happy to create a separate PR for this.

Reply:
I am actually adding an abstraction for this in a solution that is in progress for #99, so wait for that to go in, then if you want you can either modify or build on that to make an abstraction for this, I would say.